views/partials/searchCriteria.ejs

```diff
@@ -4,7 +4,7 @@
         <%- include ("labelsTags") %>
     </div>
 
-    <div class="search-criteria-knowledge-level">
+    <div id="search-criteria-knowledge-level" class="search-criteria-knowledge-level">
```

Comment:
While there's nothing wrong with having and id and a class with identical names (the browser will interpret them correctly), it does have the potential to generate confusion for yourself and other developers. So to make a favor to your future self: just name them differently.